hack/verify-golint.sh

```diff
@@ -57,28 +57,25 @@ export IFS=$'\n'
 all_packages=(
 	$(go list -e ./... | egrep -v "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_${KUBE_ROOT}/\?||")
 )
-linted_packages=(
-	$(cat $linted_file)
+failing_packages=(
+	$(cat $failure_file)
```

Comment:
While I like the idea, I think it's out of the scope of the PR and it might be tricky to implement. For instance, if line numbers change then the failure reason is now outdated.